lib/spack/spack/build_systems/aspell_dict.py

```diff
@@ -42,6 +43,17 @@ class AspellDictPackage(AutotoolsPackage):
 
     extends('aspell')
 
+    def view_destination(self, view):
```

Comment:
protocol for how to override this and when to raise exceptions should be documented someplace -- a good place is in default methods in the superclass, and probably also in the packaging guide.  But this looks good!